drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -633,6 +633,23 @@ static void set_up_fixed_clock_sources(void)
 
 		z_stm32_hsem_unlock(CFG_HW_RCC_SEMID);
 	}
+
+#if defined(RCC_HSI48_SUPPORT)
+#if defined(CONFIG_SOC_SERIES_STM32L0X)
```

Comment:
```suggestion
	if (IS_ENABLED(STM32_HSI48_ENABLED)) {
#if defined(CONFIG_SOC_SERIES_STM32L0X)
```

Reply:
this is not totally equivalent : 
- RCC_HSI48_SUPPORT if the HSI48 feature exists in the mcu
- STM32_HSI48_ENABLED if the DTS enables the existing HSI48

Reply:
To be moved inside below `if (IS_ENABLED(STM32_HSI48_ENABLED)) {` condition

Reply:
that could be 
```
#if defined(STM32_HSI48_ENABLED)
	/* For all series with HSI 48 clock support */
	if (IS_ENABLED(STM32_HSI48_ENABLED)) {
```

when `#define STM32_HSI48_ENABLED	1`  only when the node is enabled in the DTS (assuming the HSI48 exists for that device)